helm/cloud-provider-azure/Chart.yaml

```diff
@@ -13,4 +13,4 @@ maintainers:
 name: cloud-provider-azure
 sources:
 - https://github.com/kubernetes-sigs/cloud-provider-azure
-version: 1.27.7
+version: 1.27.8
```

Comment:
Since 1.27.8 is not released, please use 1.27.7 with a date as version. e.g. 1.27.7+20230814
Here is an example PR: https://github.com/kubernetes-sigs/cloud-provider-azure/pull/1745/files